starter-code/server.js

```diff
@@ -34,9 +36,13 @@ app.get('/articles', function(request, response) {
 });
 
 app.post('/articles', function(request, response) {
+  // TODO: Write a SQL query to insert a new author, ON CONFLICT DO NOTHING
   client.query(
-    '', // TODO: Write a SQL query to insert a new author, ON CONFLICT DO NOTHING
-    [], // TODO: Add the author and "authorUrl" as data for the SQL query
+    `INSERT IN author (author, authorUrl), VALUES ($1, $2) ON CONFLICT DO NOTHING;`,
```

Comment:
did you test this in the postgres console? you've got an error in your command. try comparing it to the class 9 demo code. Also, you'll need something special to make authorUrl work.